### ISSUE TITLE
Release for Safari 12

### ISSUE THREAD

**Issue by krastins-zz** [w0lfschild/SafariStand#1]
Hi, I see that your fork has some work done to support Safari 12 and Mojave, but the latest release was a year ago for Safari 11. Can we expect a new release for Safari 12? Would be greatly appreciated.

**Comment by vv2002** [comment]
Yes! It would be very nice to have SafariStand with Safari 12 and with dark theme also!

**Comment by aekiameh** [comment]
Would love to get a working release for safari 12. I miss my smaller tabs!

**Comment by xtrimo** [comment]
Me2 i wait for the release of it....

**Comment by RBrancheau** [comment]
soooo

**Comment by RBrancheau** [comment]
alright looks like safaristand is officially dead fellas. nobody is even trying to fix it for 12

**Comment by jason-hub** [comment]
What's up?  Nobody cares to update it for Mojave?  C'mon, man . . . .
